pennylane/beta/devices/default_tensor.py

```diff
@@ -165,6 +164,17 @@ def __init__(self, wires, shots=1000, representation="exact", contraction_method
         self._contraction_method = contraction_method
         self.reset()
 
+    @classmethod
+    def capabilities(cls):
+        capabilities = super().capabilities().copy()
+        capabilities.update(
+            model='qubit',
+            supports_exact=True,
+            supports_sampled=True,
+            supports_tensor_observables=True,
+        )
+        return capabilities
```

Comment:
Here the new logic is very apparent: defining  ``_capabilities`` creates a new dictionary, while overriding ``capabilities()`` this way will "inherit" all keys defined by the parent classes.